application/backend/src/main/resources/apps/typerefinery/components/widgets/search/clientlibs/style.css

```diff
@@ -0,0 +1,3 @@
+.highlight {
```

Comment:
namespace please....

Reply:
This is a global class; it is used to highlight the value. if we give it any namespace then it will work on that particular component.